rust/datafusion/DEVELOPERS.md

```diff
@@ -0,0 +1,79 @@
+# Developer's guide
```

Comment:
I pulled this file into its own separate file so that it didn't appear on https://crates.io/crates/datafusion